e2e/e2e_suite_test.go

```diff
@@ -55,7 +55,7 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 
 				<-testshell.Execute(`oc patch configs.imageregistry.operator.openshift.io/cluster --patch '{"spec":{"defaultRoute":true}}' --type=merge`).Done()
 
-				<-testshell.Execute(NewProjectCmd(GetRepositoryName())).Done()
+				<-testshell.Execute(CreateNamespaceCmd(GetRepositoryName())).Done()
```

Comment:
I think it better describes what's happening under the hood. `project` is an OpenShift concept.